build/build-sdk-images/tool/base/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN git clone --recurse-submodules -b $GRPC_RELEASE_TAG --depth 1 --shallow-subm
     mkdir -p cmake/build && \
     cd cmake/build && \
     cmake -DgRPC_INSTALL=ON -DgRPC_BUILD_TESTS=OFF ../.. && \
-    make -j && make install
+    make -j$(nproc) && make install
```

Comment:
was the `-j` not doing anything before because we weren't passing it along with a number?

Reply:
So `-j` with no number is "run with no limit of parallel processes", which can totally overwhelm a system.

Before, I would say that we just go lucky that it didn't crash the system. Or the previous debian version was limiting somehow.... which again leads to being luck. Having no limit on -j is generally just a bad idea.

Reply:
Makes sense. Thanks.